tasks/Tasks/provision_aws_cluster.robot

```diff
@@ -0,0 +1,32 @@
+*** Settings ***
+Documentation    Provision AWS Self-Managed clusters for testing Beta
+Metadata         provision
+Resource         ../Resources/Provisioning/Hive/AWS/oc-provision.robot
+Library          OperatingSystem
+Library          OpenShiftLibrary
+Library          String
+Library          Process
+
+***Variables***
+&{infrastructure_configurations}
+
+*** Tasks ***
+Provision Self-Managed Cluster In AWS &{infrastructure_configurations}
```

Comment:
Having &{infrastructure_configurations} in the task title instead of an argument is a little strange.  What is the benefit of it? Is this the only way to load it when is been created as a variable file?